libs/browser/src/lib/services/browser-micro-sentry-client.ts

```diff
@@ -12,12 +12,14 @@ import { State } from '../models/state';
 import { MicroSentryPlugin } from '../models/plugin';
 import { BrowserSentryClientOptions } from '../models/browser-sentry-client-options';
 import { isMatchingPattern } from '../utils/is-matching-pattern';
+import { MAX_BREADCRUMBS } from '../consts/max-breadcrumbs';
 
 function getWindow(): Window {
   return window;
 }
 
 export class BrowserMicroSentryClient extends MicroSentryClient {
+  private readonly breadcrumbsKeyName = 'breadcrumbs';
```

Comment:
Do we need this property
Maybe use dot notation (.breadcrumbs) in property access instead?

Reply:
you mean `get breadcrumbsKeyName()` ?
it seems it requires additional type adjustments which looks more complicated

Reply:
nope, i meant use just { breadcrumbs: ... } instead of { ['breadcrumbs']: ... }
but that's nit i guess